components/vf-card/vf-card.scss

```diff
@@ -17,16 +17,17 @@
 
   background-color: var(--card-background-color);
   display: grid;
-  grid-template-rows: 216px 1fr;
+  grid-template-rows: 288px 1fr;
   position: relative;
 }
 
 .vf-card__image {
   grid-column: 1 / -1;
-  grid-row: 1 / span 3;
+  grid-row: 1;
   height: 100%;
   max-width: 100%;
   object-fit: cover;
+  object-position: center;
```

Comment:
When we have Safari/IE/etc specific fixes, I think it would be good if we had a standardised way to note the fix; something like

```css
  object-fit: cover;
  object-position: center; // safari centring fix: https://github.com/visual-framework/vf-core/pull/1087
  width: 100%;
}
```

Reply:
`object-position: center` will work for all browsers [that support it](https://caniuse.com/#feat=mdn-css_properties_object-position).

The issue with Safari was having `grid-column: 1 / span 3` on the `vf-card__image` (and I cannot see a reason why it's there) because it was making the grid row for the image 3 x 216px tall. It didn't do this in other browsers (I'm guessing Gecko and Chromium work out the image isn't that large and makes it 'as big as it actually is'.

So I've bumped up the height of the first row to make the `vf-card__image` as tall as it was in other browsers.

So this works 'as expected' across all browsers except IE11 - which just defaults to letting the content flow.

Reply:
but yes, specific browser bug fixes should get a nice little comment so we know why it's there.

Reply:
in this case it was more 'bad CSS' than 'bad browser' :D

Reply:
Sounds good :)